### ISSUE TITLE
[Docs] Model export/conversion guide.

### ISSUE THREAD

**Issue by Y-T-G** [OML-Team/open-metric-learning#491]
It would be nice to have some model export guides for deployment.

**Comment by AlekseySh** [comment]
Hey, @Y-T-G! 
What models (architectures) are you particularly interested in?
Are we talking about converting to ONNX format?

**Comment by AlekseySh** [comment]
Hey, sorry for the late response. I thought you might have had trouble converting one of the more complex architectures to ONNX, like with some CLIP models, due to how their weights were jitted. But for vanilla ResNet, you shouldn't face any issues converting it to ONNX. This function isn't directly connected to metric learning, so there are no specific instructions for it.

You google working with ONNX by yourself or start with this tiny script and report problems if any:
```
import torch
import torchvision.models as models
import torch.onnx

# Load the pre-trained ResNet50 model
model = models.resnet50(pretrained=True)

# Set the model to evaluation mode
model.eval()

# Create a dummy variable with the correct shape 
# The shape for ResNet is typically (batch_size, 3, 224, 224)
x = torch.randn(1, 3, 224, 224, requires_grad=True)

# Define the output path for the ONNX model
output_onnx = 'resnet50.onnx'

# Export the model to ONNX format
torch.onnx.export(model,               # model being run
                  x,                   # model input (or a tuple for multiple inputs)
                  output_onnx,         # where to save the model
                  export_params=True,  # store the trained parameter weights inside the model file
                  opset_version=10,    # the ONNX version to export the model to
                  do_constant_folding=True,  # whether to execute constant folding for optimization
                  input_names = ['input'],   # the model's input names
                  output_names = ['output'], # the model's output names
                  dynamic_axes={'input' : {0 : 'batch_size'},    # variable length axes
                                'output' : {0 : 'batch_size'}})

```
